<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TouchScreen</class>
 <widget class="QDialog" name="TouchScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>0</height>
    </size>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>194</red>
         <green>198</green>
         <blue>189</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>194</red>
         <green>198</green>
         <blue>189</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>194</red>
         <green>198</green>
         <blue>189</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabBar::tab { height: 70px; width: 400px; }</string>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>23</height>
    </size>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab">
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>MainMenu</string>
    </attribute>
    <widget class="QPushButton" name="systemOffPushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>371</width>
       <height>321</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#systemOffPushButton {
background-color: transparent;
border-image: url(:MyFiles/PowerOff.gif);
background: none;
border: none;
background-repeat: none;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="edfaOffPushButton">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>150</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#edfaOffPushButton{
background-color: transparent;
border-image: url(:MyFiles/Red.png);
background: none;
border: none;
background-repeat: none;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>150</y>
       <width>141</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Amplifier:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="laserOffPushButton">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>50</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#laserOffPushButton{
background-color: transparent;
border-image: url(:MyFiles/Red.png);
background: none;
border: none;
background-repeat: none;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>40</y>
       <width>141</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Laser:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>260</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Stable:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stableOnPushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>250</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#stableOnPushButton {
background-color: transparent;
border-image: url(:MyFiles/Green.png);
background: none;
border: none;
background-repeat: none;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="laserOnPushButton">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>50</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#laserOnPushButton {
background-color: transparent;
border-image: url(:MyFiles/Green.png);
background: none;
border: none;
background-repeat: none;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="edfaOnPushButton">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>150</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#edfaOnPushButton {
background-color: transparent;
border-image: url(:MyFiles/Green.png);
background: none;
border: none;
background-repeat: none;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="stableOffPushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>250</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#stableOffPushButton{
background-color: transparent;
border-image: url(:MyFiles/Red.png);
background: none;
border: none;
background-repeat: none;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="systemOnPushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>371</width>
       <height>321</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#systemOnPushButton {
background-color: transparent;
border-image: url(:MyFiles/PowerOn.gif);
background: none;
border: none;
background-repeat: none;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="laserPortOffPushButton">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>50</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#laserPortOffPushButton{
background-color: transparent;
border-image: url(:MyFiles/Red.png);
background: none;
border: none;
background-repeat: none;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="edfaPortOffPushButton">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>150</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#edfaPortOffPushButton {
background-color: transparent;
border-image: url(:MyFiles/Red.png);
background: none;
border: none;
background-repeat: none;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="laserPortOnPushButton">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>50</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#laserPortOnPushButton{
background-color: transparent;
border-image: url(:MyFiles/Green.png);
background: none;
border: none;
background-repeat: none;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="edfaPortOnPushButton">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>150</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#edfaPortOnPushButton{
background-color: transparent;
border-image: url(:MyFiles/Green.png);
background: none;
border: none;
background-repeat: none;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>20</y>
       <width>221</width>
       <height>33</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Emission:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <zorder>layoutWidget</zorder>
    <zorder>edfaPortOnPushButton</zorder>
    <zorder>edfaPortOffPushButton</zorder>
    <zorder>laserPortOnPushButton</zorder>
    <zorder>systemOnPushButton</zorder>
    <zorder>edfaOnPushButton</zorder>
    <zorder>laserOnPushButton</zorder>
    <zorder>systemOffPushButton</zorder>
    <zorder>edfaOffPushButton</zorder>
    <zorder>label_2</zorder>
    <zorder>laserOffPushButton</zorder>
    <zorder>label</zorder>
    <zorder>label_7</zorder>
    <zorder>stableOnPushButton</zorder>
    <zorder>stableOffPushButton</zorder>
    <zorder>laserPortOffPushButton</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Settings:</string>
    </attribute>
    <widget class="QPushButton" name="savePushButton">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>310</y>
       <width>131</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#savePushButton {
background-color: transparent;
border-image: url(:MyFiles/Save.gif);
background: none;
border: none;
background-repeat: none;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>731</width>
       <height>252</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>EDFA Power(dBm):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="powerReadoutSpinBox">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>36</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="maximum">
           <double>999.990000000000009</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="powerSetpointSpinBox">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>36</pointsize>
           </font>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::PlusMinus</enum>
          </property>
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="maximum">
           <double>30.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Wavelength(nm):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="wavelengthReadoutSpinBox">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>36</pointsize>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>9999.998999999999796</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="wavelengthSetpointSpinBox">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>36</pointsize>
           </font>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::PlusMinus</enum>
          </property>
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>1559.765000000000100</double>
          </property>
          <property name="maximum">
           <double>1560.886999999999944</double>
          </property>
          <property name="singleStep">
           <double>0.001000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="loadDefaultPushButton">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>327</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset To Saved</string>
     </property>
    </widget>
    <zorder>layoutWidget</zorder>
    <zorder>savePushButton</zorder>
    <zorder>loadDefaultPushButton</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
